### PR TITLE
Add Fleet/Agent release notes for 8.1.2

### PR DESCRIPTION
**Pull request by dedemorton**

### CODE REVIEW

**Review by jen-huang — APPROVED** [review]
Fleet (no content 😄) LGTM

File: docs/en/ingest-management/release-notes/release-notes-8.1.asciidoc [hunk 1]
```diff
@@ -20,6 +21,26 @@ Also see:
...
+[[bug-fixes-8.1.2]]
+=== Bug fixes
+
+{fleet}::
+* There are no bug fixes for {fleet} in this release.
+
+{agent}::
+* Propagate input ID from the Agent policy into the Filebeat configuration. Note
+that no validation is performed on this field. {agent-pull}30386[#30386]
+
```
Comment (ph) [inline docs/en/ingest-management/release-notes/release-notes-8.1.asciidoc:41]: * Fix the start sequence of Beats that was non-deterministic making Beats missing their 
configuration from Agent and not sending events. {agent-pull}30694[#30694]

**Review by ph — CHANGES_REQUESTED** [review]

File: docs/en/ingest-management/release-notes/release-notes-8.1.asciidoc (hunk 1 above)
Comment (ollyhowell) [reply docs/en/ingest-management/release-notes/release-notes-8.1.asciidoc:41]: Adding this suggestion

**Review by ollyhowell — APPROVED** [review]
Looks like my addition builds ok so merging...

**Review by ph — COMMENTED** [review]
LGTM post-merge no worries @ollyhowell

**Pull request merged**